src/selectors/index.ts

```diff
@@ -0,0 +1,12 @@
+import { createSelector } from 'reselect';
+import { Comment } from '../state/comments'
+import { State } from '../state'
+
+const getComments = (state: State) => state.comments.comments;
+
+export function selectCommentsForContentPathFactory (contentPath: string) {
+    return createSelector(
+        getComments,
+        (comments: Map<number, Comment>) => [...comments.values()].filter((comment: Comment) => comment.contentPath === contentPath)
```

Comment:
**Question:** ` [...comments.values()]`  <-- Doesn't filter work on `comments.values()`?

Reply:
Unfortunately no, it wants an array for `filter`